low-level/files/opt/seagate/sspl/setup/resource_map/resource_map.py

```diff
@@ -35,3 +37,37 @@ def get_health_info(self, rpath):
         rpath: Resource id (Example: hw>disks)
         """
         pass
+
+    def get_data_template(self, uid: str, is_fru: bool):
+        """Get the health data template."""
+        return {
+            "uid": uid,
+            "fru": str(is_fru).lower(),
+            "last_updated": -1,
+            "health": {
+                "status": "",
+                "description": "",
+                "recommendation": "",
+                "specifics": []
+            }
+        }
+
+    def set_health_data(self, obj: dict, status, description=None,
+                        recommendation=None):
+        """Set the given or default health data as per health status."""
+        good_state = (status == "OK")
```

Comment:
Please give a sample input.

Reply:
In new PR using the already present method in health branch, added by Jeet.

Reply:
In that case, please rebase to avoid code duplication.